date.go

```diff
@@ -132,10 +132,18 @@ func (d *dateObject) unset() {
 }
```

Comment:
[q] do we need to set `d.val.__wrapped` to `nil` in the `unset()` and `setTimeMs` functions above? I could be missing something, but we might need to take into account if `d.val.__wrapped` is set in a few other places where we use `d.msec` within this file and `builtin_date.go`.

Reply:
excellent callout and catch. I updated to reflect that.